Makefile

```diff
@@ -116,6 +116,9 @@ e2e_scenarios:
 	./scripts/tests/start_env.sh
 	./scripts/tests/force_clean_env.sh
 	./create/tests/new.sh
-	./create/tests/demos/btc_eth.sh
+	./create/tests/force_clean_env.sh
```

Comment:
There is a `wait` and `kill` at the end of each test that prevents the need of such added file.

Reply:
Which does not seem to work 100%. 
I think we should resolve #388 and then use this here.